FrameRate/FrameRate.cpp

```diff
@@ -266,13 +266,17 @@ namespace WPEFramework
             LOGINFOMETHOD();
             returnIfParamNotFound(parameters, "framerate");
 
+	    bool hasPersist = parameters.HasLabel("persist");
+	    bool persist = hasPersist ? parameters["persist"].Boolean() : false;
+	    if (!hasPersist) LOGINFO("persist: false");
```

Comment:
you should print in both true and false case

Reply:
added print in else case for persist as true
if (!hasPersist) LOGINFO("%s persist: false",__FUNCTION__); else LOGINFO("%s persist: true",__FUNCTION__);